geps/gep-1713/index.md

```diff
@@ -115,6 +115,16 @@ type ListenerSetSpec struct {
 	// 2. ListenerSet ordered by creation time (oldest first)
 	// 3. ListenerSet ordered alphabetically by “{namespace}/{name}”.
 	//
+	// Regarding Conflict Management, Listeners in a ListenerSet follow the same
+	// rules of Listeners on a Gateway resource.
+	// 
+	// This validation should happen within all of the ListenerSets attached to a 
```

Comment:
I think there should be two sections - (i) validation of listeners in a listenerSet (ii) listenerSets in a gateway with examples
Reading this, I would think that all listeners should be validated together as is done in a Gateway